run_all_experiments.py

```diff
@@ -322,6 +324,10 @@ def _print_experiment_results(results: list[Result]):
         f'*{result.benchmark.project}, {result.benchmark.function_signature}*'
         f'\n{result.result}\n')
 
+  print('\n\n**** TOTAL COVERAGE GAIN: ****\n\n')
+  for project in cov_gain:
+    print(f'*{project}: {cov_gain[project]}')
```

Comment:
Use the newly unified logging, please.
https://github.com/google/oss-fuzz-gen/pull/491#discussion_r1681677693